core/store/src/main/java/uk/gov/gchq/gaffer/store/library/GraphLibrary.java

```diff
@@ -327,44 +327,47 @@ private void nullCheck(final String graphId, final Schema schema, final StorePro
 
 
     public Schema resolveSchema(final Schema schema, final List<String> parentSchemaIds) {
-        Schema resultSchema = null;
+        Schema.Builder rtn = new Schema.Builder();
```

Comment:
Why is this `rtn`. It's not being returned, one of it's methods run and the result of that is returned. Even if it was being returned, `rtn` is not a good name.